packages/riipen-ui/src/components/Chip.jsx

```diff
@@ -198,85 +209,137 @@ class Chip extends React.Component {
             border-color: ${theme.palette.primary.main};
             color: ${theme.palette.primary.contrast};
           }
+          .primary.hover:hover {
```

Comment:
Not sure if we want to darken chips on hover, thoughts?

Reply:
I was thinking that _outlined_ chips would become _default_ and _default_ would become _outlined_. 

But that also depends on what the designs specify, I would ask Maribel to decide what she wants to do in this situation

Reply:
I shall ask her, convert this PR to draft for now, and then update when ready!

Reply:
Updated to be the inverted variant!